galsim/phase_screens.py

```diff
@@ -448,11 +455,18 @@ def _zern_rho_coefs(n, m):
     A[n-2*kmax] = val
     return A
 
-
-def _zern_coef_array(n, m, eps=0., shape=None, annular=False):
+def _zern_coef_array(n, m, eps, shape, annular):
     """Assemble coefficient array for evaluating Zernike (n, m) as the real part of a
     bivariate polynomial in abs(rho)^2 and rho, where rho is a complex array indicating position on
     a unit disc.
+
+    @param n        Zernike radial coefficient
+    @param m        Zernike azimuthal coefficient
+    @param eps      Linear obscuration fraction.
```

Comment:
Yep.  Mahajan calls this epsilon, which is why I went with `eps` in the lower-level functions.

I'll have a chance to look through the rest of this (and the other pending PRs) tomorrow...